assets/sass/globals/accents.scss

```diff
@@ -10,8 +10,6 @@
     border-top-style: solid;
 }
 
-.a--border-left {  }
```

Comment:
If a selector is deleted without ever having any content, was it ever really CSS?